### PR TITLE
Add troubleshooting for Docker

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

**Comment by kkraune** [comment]
ping @ thomasht86

File: en/operations-selfhosted/docker-containers.html [hunk 1]
```diff
@@ -311,3 +311,23 @@ <h2 id="resource-usage">Resource usage</h2>
...
+
+
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+<p>
+  The Vespa documentation examples use <code>docker</code>.
+  The Vespa Team has good experience with using <code>podman</code>, too,
+  in the examples just change from <code>docker</code> to <code>podman</code>.
+  <a href="https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman">emulating-docker-cli-with-podman</a>
+  is a useful resource.
```
Comment (thomasht86) [inline en/operations-selfhosted/docker-containers.html:323]: Maybe add a note that Podman v5 is recommended for MacOS (uses Apple native hypervisor, which is said to perform much better), or if Nvidia GPU support is desired. https://github.com/containers/podman/blob/main/RELEASE_NOTES.md

**Review by thomasht86 — APPROVED** [review]
Sorry, this went under the radar. 
Thanks for reminder.
Good addition to troubleshooting. 
Consider adding details on Podman v5.

File: en/operations-selfhosted/docker-containers.html (hunk 1 above)
Comment (kkraune) [reply en/operations-selfhosted/docker-containers.html:323]: Good point! added

**Pull request merged**